ksqldb-common/src/test/java/io/confluent/ksql/function/UdfIndexTest.java

```diff
@@ -964,6 +936,60 @@ public void shouldThrowIfNoExactMatchAndImplicitCastDisabled() {
             + "(INTEGER)"));
   }
 
+  @Test
+  public void shouldThrowWhenGivenVagueImplicitCast() {
+    // Given:
+    givenFunctions(
+        function(FIRST_FUNC, false, LONG, LONG),
+        function(SECOND_FUNC, false, DOUBLE, DOUBLE)
+    );
+
+    // When:
+    final Exception e = assertThrows(Exception.class,
```

Comment:
Is the type of exception thrown really just Exception rather than something more specific?

Same question for the other new negative test below.

Reply:
I just noticed other tests used Exception. I could make it a KsqlException with no issues, seems to work fine.